src/constants/commands.ts

```diff
@@ -28,6 +28,19 @@ export const MENTION_EACH = {
   ],
 };
 
+export const USER = {
+  name: "user",
+  description: "Replies with specific user details in the channel",
+  options: [
+    {
+      name: "username",
+      description: "Takes username and shows user details",
+      type: 6, // 2 is type SUB_COMMAND_GROUP
```

Comment:
why type is hard coded

Reply:
Mandatory field with respect to the command I have built.
Reference : 
- https://discord.com/developers/docs/interactions/application-commands
![image](https://github.com/Real-Dev-Squad/discord-slash-commands/assets/53934353/f285be28-0d29-49ff-9f04-04a36fe6c945)